testing/environments/docker/kibana/Dockerfile-snapshot

```diff
@@ -6,7 +6,9 @@ ARG DOWNLOAD_URL
 ARG ELASTIC_VERSION
 ARG XPACK
 
-HEALTHCHECK --retries=6 CMD curl -f http://localhost:5601
+RUN apt-get update && apt-get install -y jq && apt-get clean
```

Comment:
This will probably slow down our builds quite a bit. Is that something we should contribute back to the official image?